spec/mongoid/association/relatable_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Mongoid::Association::Relatable do
+
+  context "anonymous classes" do
+    let(:klass) do
+      Class.new do
+        include Mongoid::Document
+        embeds_many :addresses
+      end
+    end
+
+    it "allows relations" do
```

Comment:
This should be  something like 

```
context 'when an association is defined' do
  it 'loads successfully' do
```

Also, this is an integration test, it should be under `spec/integration` probably in `spec/integration/associations` and should have a label like "when an association is defined in an anonymous class" ...